scipy/optimize/_root.py

```diff
@@ -684,8 +684,8 @@ def _root_krylov_doc():
 
         rdiff : float, optional
             Relative step size to use in numerical differentiation.
-        method : {'lgmres', 'gmres', 'bicgstab', 'cgs', 'minres'} or function
-            Krylov method to use to approximate the Jacobian.
+        method : {'lgmres', 'gmres', 'bicgstab', 'cgs', 'minres', 'tfqmr'} or
+            function Krylov method to use to approximate the Jacobian.
```

Comment:
This change seems incorrect, since line 687 is the type descriptor and 688 the textual description. Instead, " or function" should be "or callable". And then perhaps you want to clarify the description in some other way.

Reply:
Thanks, you are right. I'll fix it.